patch-setup-config.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
```

Comment:
Yes, but `fixFile` can't handle binary file patching cleanly (we need to consider I/O mode, encoding etc. problems), so I made a temporary hack.